go.mod

```diff
@@ -3,96 +3,128 @@ module github.com/CosmWasm/wasmd
 go 1.18
 
 require (
+	cosmossdk.io/errors v1.0.0-beta.7
+	cosmossdk.io/math v1.0.0-beta.3
+	cosmossdk.io/simapp v0.0.0-20221107225151-b49f948b36bc
 	github.com/CosmWasm/wasmvm v1.1.1
-	github.com/cosmos/cosmos-proto v1.0.0-alpha7
-	github.com/cosmos/cosmos-sdk v0.45.9
+	github.com/cosmos/cosmos-proto v1.0.0-alpha8
+	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20221107225151-b49f948b36bc
```

Comment:
This is main branch, right?

We definitely want to get a proper CI and testing on this once this is tagged (as an rc) along with ibc-go.
Thanks for having an eye on it... I hear some rc tags are coming soon....